apps/src/templates/sectionsRefresh/SectionsSetUpContainer.jsx

```diff
@@ -171,7 +171,7 @@ export default function SectionsSetUpContainer({
       : null;
 
     const computedGrades =
-      participantType === 'teacher' ? ['pl'] : section.grade;
+      participantType === 'student' ? section.grade : ['pl'];
```

Comment:
not a big deal but why change this?

Reply:
Actually I think it might have been a bigger issue! I forgot until troubleshooting these tests that there can be classes of facilitators, so switching this means that I'm catching any other non-student cases